bears/haskell/HaskellLintBear.py

```diff
@@ -38,11 +38,15 @@ def process_output(self, output, filename, file):
         output = json.loads(output)
 
         for issue in output:
-            assert issue['startLine'] == issue['endLine']
             diff = Diff(file)
+            from_lines = issue['from'].splitlines()
+            to_lines = issue['to'].splitlines()
+            assert len(from_lines) == len(to_lines)
```

Comment:
The `patch` that we apply for correcting the error is only the for the first line of the whole multi line message. From what I have seen, the whole body ( that is message structure ) is the same except the first line where we actually make changes.

These assert statements are required to check that the whole message body ( structure ) is same, and that only the first line needs a patch ( diff ).

Otherwise the patch cannot be applied if there are multiple differences in the `from` and `to`, i.e actual code and linter suggested code.

I guess it'll be better understood once I add a proper testcase?!